### ISSUE TITLE
Change the globus message for onboarding DCCs?

### ISSUE THREAD

**Issue by ACharbonneau** [nih-cfde/security#14]
Is it possible to change the language of this pop up box?

![image](https://user-images.githubusercontent.com/1719360/110695667-28003580-81b8-11eb-94ac-e07b09f40f42.png)

Users are confused about where they are supposed to click. It's not intuitive to have to "apply for membership" when they were invited to a group

**Comment by lliming** [comment]
Yes, that is non-intuitive. I'll report this to Globus as a feature change request and keep this thread posted as I hear back.

**Comment by lliming** [comment]
Commenting here for my own convenience... The Globus ticket # for this is 350518.
